map/bookmark_manager.hpp

```diff
@@ -373,6 +373,10 @@ class BookmarkManager final
   bool SaveBookmarkCategory(kml::MarkGroupId groupId);
   bool SaveBookmarkCategory(kml::MarkGroupId groupId, Writer & writer, KmlFileType fileType) const;
 
+  bool HasRecentlyDeletedBookmark() const { return m_recentlyDeletedBookmark != nullptr; };
```

Comment:
nit: Does `return m_recentlyDeletedBookmark;` work? C++ smart pointers (unique_ptr, shared_ptr) have overloaded `operator bool` that returns `true` if a smart pointer is initialized.

Reply:
Yes it works... This method works well and always returns the correct bool value... How I should change the code?

Reply:
It's a nit, leave as is or change to:
```suggestion
  bool HasRecentlyDeletedBookmark() const { return m_recentlyDeletedBookmark; };
```

Reply:
ACK. The right syntax is not as readable, so let's leave it as is:
```
return ptr.operator bool();
```

Reply:
Thanks! Fixed.